### PR TITLE
Redesign Sensor widget

### PR DESCRIPTION
**Pull request by Penait1**
## Summary
Redesign of the Sensor widget

## Screenshots
Screenshot 2024-11-20 at 17 04 05
Screenshot 2024-11-19 at 19 52 17

### CODE REVIEW

**Comment by Penait1** [comment]
@bgoncal I think its ready for review, the test step seems to be failing but I'm not sure why

File: Sources/Extensions/Widgets/Common/WidgetBasicContainerView.swift [hunk 1]
```diff
@@ -86,22 +88,26 @@ struct WidgetBasicContainerView: View {
...
                 HStack(spacing: Spaces.one) {
                     ForEach(column) { model in
                         if case let .widgetURL(url) = model.interactionType {
                             Link(destination: url.withWidgetAuthenticity()) {
                                 if #available(iOS 18.0, *) {
-                                    WidgetBasicViewTintedWrapper(model: model, sizeStyle: sizeStyle)
+                                    tintedWrapperView(model: model, sizeStyle: sizeStyle)
                                 } else {
-                                    WidgetBasicView(model: model, sizeStyle: sizeStyle, tinted: false)
+                                    WidgetBasicButtonView(model: model, sizeStyle: sizeStyle, tinted: false)
```
Comment (bgoncal) [inline Sources/Extensions/Widgets/Common/WidgetBasicContainerView.swift]: Should this case always be a button view?

**Review by bgoncal — CHANGES_REQUESTED** [review]
Just tested on my device and it looks great! One minor thing, the widget background needs to be that gray-ish that the other widgets before, because we try tor eplicate the lovelace UI, gray-ish bg + white background tile, I think the design I shared with you had the wrong bg as well, my bad.

Also are you planning to add more sizes? (can be in a separate PR of course)

File: Sources/Extensions/Widgets/Common/WidgetBasicContainerView.swift (hunk 1 above)
Comment (Penait1) [reply Sources/Extensions/Widgets/Common/WidgetBasicContainerView.swift]: Mm no I was mistaken

**Comment by Penait1** [comment]
Yes I'm planning more sizes but I'll do it in a separate pr. 

I quickly checked if the sensor type view works with bigger sizes and it works. I'll have to do some extra stuff for the snapshot preview still I think

**Review by bgoncal — APPROVED** [review]

**Pull request merged**